sqlmesh/dbt/model.py

```diff
@@ -423,6 +439,80 @@ def to_sqlmesh(self, context: DbtContext) -> Model:
                     "target_lag": self.target_lag,
                 }
 
+        if context.target.dialect == "clickhouse":
+            unsupported_materializations = [
+                "dictionary",
+                "distributed_table",
+                "distributed_incremental",
+                "materialized_view",
+            ]
+            if self.materialized in unsupported_materializations:
+                fallback = self.materialized.split("_")
+                msg = f"SQLMesh does not support the '{self.materialized}' model materialization."
+                if len(fallback) == 1:
+                    # dictionary materialization
+                    raise ConfigError(msg)
+                else:
+                    logger.warning(f"{msg} Falling back to the '{fallback[1]}' materialization.")
+                    self.materialized = fallback[1]
+
+            if self.model_materialization == Materialization.INCREMENTAL:
+                if self.incremental_strategy in ["delete+insert", "append", "insert_overwrite"]:
+                    logger.warning(
+                        f"The '{self.incremental_strategy}' incremental strategy is not supported - SQLMesh uses the temp table/partition swap strategy for all incremental models."
```

Comment:
How about the `INCREMENTAL_UNMANAGED` type? Can't we support `append` and `insert_overwrite` with it?

Reply:
Excellent point